addons/project/static/src/project_sharing/project_sharing.js

```diff
@@ -34,6 +35,9 @@ export class ProjectSharingWebClient extends Component {
 
     async _showView() {
         const { action_name, project_id, open_task_action } = session;
+        if (action_name.help) {
+            action_name.help = markup(escape(action_name.help));
```

Comment:
Sorry, in fact, it is not a good idea to use `escape` since it will interpret the helper as a string instead of html

Reply:
Okay, Done!!